tasks/kubectl-apply-service-and-statefulset-no-patch.yml

```diff
@@ -0,0 +1,46 @@
+platform: linux
```

Comment:
I can't see this file being used anywhere

Reply:
Fixed